facilitation_guide/session04_script.Rmd

```diff
@@ -58,8 +58,55 @@ Let’s go over decisions and actions completed since last review. {insert for e
 <img src = "https://raw.githubusercontent.com/lzim/teampsd/hexagon_icons/np_synchronize_778914_003F72.png"
      height = "150" width = "150">
 
-### Do
+### Do (30 minutes)
 [Go over the session Overview, Learning Objectives, and Agenda.]
+ 
+
+Describe the factors controlling the relationship between the number of patients in care and the backlog of appointments on the schedule. (5 minutes)
+
+In a discussion, get the group to delve into:
+
+Decision process for booking a current patient’s next visit (who, when, how, why, etc.)
+How far out are they booking right now (look for any conflict between provider’s desired time for their patient’s next visit and the actual time of that next visit)
+
+Facilitator Goal: Group understands the difference between the number of patients they have in care (stock), and the number of appointments they see per week (flow), so even if every patient has an appointment on the schedule, not every patient is seen every week. 
+
+In-depth explanation: There is a key simplifying difference between the model and reality -- The model has no "appointment calendar," so there can be some difficulty understanding and accepting how the model keeps track of appointments. In the real world, usually each patient only has one appointment in the calendar at any given time.  Each appointment is created as soon as their current appointment has passed, at the desired return visit interval.  Since the model is of the entire patient cohort, it doesn't track individual patients, or their individual appointments.  Instead, the entire cohort generates a certain number of appointments each week, based on the RVI. So, if the RVI is every week, then the entire cohort will get an appointment booked for them every week. If the RVI is every month, then only 1/4 of the cohort will get an appointment booked each week.  These appointments accumulate in the backlog of appointments to be seen, or more simply "Appointment Backlog." The same logic applies to appointment completions: if the RVI is 1 week, then the team will complete all appointments in the backlog each week (if there is enough supply to do it).  If the RVI is 1 month, then the team will only complete 1/4 of the appointments in the backlog each week (again, if there is enough supply).
```

Comment:
This is great! I want to make a note to revisit just to make sure the 2nd half of the paragraph continues to make clear that it's describing how the MODEL works (e.g., "...then the entire cohort will get an appointment booked for them every week." Not sure how to do this kind of tweaking efficiently with GitHub but will at least make note.